runtest.py

```diff
@@ -528,6 +539,19 @@ def footer(self, f):
 if pythonpath:
     testenv['PYTHONPATH'] = testenv['PYTHONPATH'] + os.pathsep + pythonpath
 
+if sys.platform == 'win32':
+    # Windows doesn't support "shebang" lines directly (the Python launcher
+    # and Windows Store version do, but you have to get them launched first)
+    # so to directly launch a script we depend on an assoc for .py to work.
+    # Some systems may have none, and in some cases IDE programs take over 
+    # the assoc.  Detect this so the small number of tests affected can skip.
+    try:
+        python_assoc = get_template_command('.py')
+    except OSError:
+        python_assoc = None
+    if not python_assoc or "py" not in python_assoc:
```

Comment:
What's python_assoc?

Reply:
made-up local variable for the looked-up association to the Python filetype.

Reply:
ahh.. always have coffee before reviewing.. ;)